app/src/main/java/net/opendasharchive/openarchive/features/settings/ProofModeSettingsActivity.kt

```diff
@@ -67,8 +79,9 @@ class ProofModeSettingsActivity: BaseActivity() {
 
             val pkePreference = findPreference<SwitchPreferenceCompat>(Prefs.USE_PROOFMODE_KEY_ENCRYPTION)
             val activity = activity
+            val availability = Hbks.deviceAvailablity(requireContext())
```

Comment:
Just out of curiosity: Why do you use `requireContext()` (which makes the app crash) here instead of the activity from one line above and do the null check before?